src/pods/edit-realtion/edit-relation.component.tsx

```diff
@@ -1,3 +1,47 @@
-export const EditRelation: React.FC = () => {
-  return <></>;
+import React from 'react';
+import { Dropdown, DropdownOptionVm } from '@/common/components/dropdown';
+import { RelationType, RelationVm } from '@/core/providers/canvas-schema';
+
+interface Props {
+  relationsTipeOptions: DropdownOptionVm[];
+  tablesNameOptions: DropdownOptionVm[];
+  relation: RelationVm;
```

Comment:
relation should be optional? When you add a Relation this field is null?

```ts
relation?: RelationVm;
```

Reply:
No, because the 'edit-relation.component' always needs to receive a 'relation' object. This object serves as a container to store data from the selects, and it's passed to the 'edit-relation.pod.' When changes are applied, I send the updated 'relation' object.